packages/flutter/lib/src/widgets/editable_text.dart

```diff
@@ -572,6 +576,18 @@ class EditableText extends StatefulWidget {
   /// Defaults to false. Cannot be null.
   /// {@endtemplate}
   final bool obscureText;
+  
+  /// {@template flutter.widgets.editableText.obscureTextBehavior}
+  /// Whether or not the last character entered is shown before
+  /// being obscured.
+  /// When this is set to [ObscureTextBehavior.all] or [ObscureTextBehavior.delayed], the last characters in the text field is
+  /// replaced by [obscuringCharacter].
```

Comment:
This is probably too specific, maybe change it to:

```
How characters in the field are obscured, if at all.

For example, a password field may want to obscure the entered text so it's not readable.

When this is set to [ObscureTextBehavior.all] or [ObscureTextBehavior.delayed], the characters in the field are replaced by [obscuringCharacter].
```

But make sure that lines don't exceed 80 characters in length.